### DISCUSSION TITLE
SeeedStudio BeagleBone® Green | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1682] [Q&A]
# SeeedStudio BeagleBone® Green | Seeed Studio Wiki

SeeedStudio BeagleBone® Green

https://wiki.seeedstudio.com/BeagleBone_Green/

**Comment by waterwin** [comment]
You might want to correct this : Mechincal Drawing

**Comment by lilMochi** [comment]
Is the C162 capacitor still being used? I dont see it on the physical board anymore

**Reply by HM-sur** [reply]
We have reviewed the schematic of the historical versions and did not find the C162 capacitor. 
Could you please let us know where you saw this version with the C162 capacitor? 
We look forward to your response.

**Reply by lilMochi** [reply]
i see it on my BBG2, also it is here on the wiki 

"2. Sometimes the Seeed Studio BeagleBone® Green's network port does not work and must be restarted to resume work. Occasionally this happens.

Please replace a stable power supply and try again. Usually mobile phone USB power adapter is more stable than that computer USB. Or you can remove the capacitor C162"

is the BBG2 still in production? I just order a unit and got a GGB1 which is the lastest build ?

**Reply by HM-sur** [reply]
I apologize for the error in our documentation. We are currently working to correct it. The BBG was updated in 2021, and the C162 mentioned in the content is no longer available. Thank you for your understanding.

**Reply by lilMochi** [reply]
Understood, thank you. I have both BBG1 and BBG2 could you please tell me what are the difference? I see a difference in the components and the layout is slightly different. is BBG2 in production. i just order it from seeed studio directly and got BBG1

**Reply by HM-sur** [reply]
According to FAQ1, there is a difference in eMMC between BBG1 and BBG2, but after using our new firmware, both BBG1 and BBG2 can operate normally and are not affected

**Reply by lilMochi** [reply]
just trying to understand the difference of the 2. is the BBG2 a have a newer layout of the components? i see a difference in many part of the board

**Reply by ReKST-117** [reply]
Hi, I see some components changes between these two schematics so I am assuming it is a new layout for BBG2.

**Reply by lilMochi** [reply]
I just purchased few units from Seeed directly and they sent me BBG1 so i wasnt sure if this was the new revision , seems like they are still selling them to buyers.

**Reply by ReKST-117** [reply]
Actually, there is no functionality difference between BBG1 and BBG2 so it should be fine.

**Reply by lilMochi** [reply]
Im uping the P9 5, and 6 to power up the unit, some time it would not start up the ETH PHY and have to power cycle the units. is there a way to get this to power up properly?

**Reply by nfs0619** [reply]
Thanks for bringing this to our attention. We’ve recorded your feedback and will arrange to reproduce and investigate this in our testing process, which may take some time.

In the meantime, we recommend sharing more details (such as power supply specs, startup sequence, and firmware version) with our technical support team at [techsupport@seeed.io](mailto:techsupport@seeed.io), as they may be able to provide quicker and more detailed guidance.

**Reply by lilMochi** [reply]
could you also please update the altium model and all the resource here to  V1.2. is it V1

**Reply by bleaaach** [reply]
Hi, currently only version v1.1 is available; version v1.2 is not yet released.
